### DISCUSSION TITLE
Pages in user's profile repo should not contain it's own namespace

### DISCUSSION THREAD

**Discussion by mateusabelli** [community/community#50549] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I have the following repository https://github.com/mateusabelli/mateusabelli to provide a README on my profile, but I would also like to host a personal portfolio in there and the pages deployment looks like this https://mateusabelli.github.io/mateusabelli/.

I think the URL in this format https://mateusabelli.github.io/ would look better, since Pages is deploying from my special profile repository I don't think it should contain it's own namespace at the end. 

I feel like we could maybe implement this feature or just a redirect, so people can host portfolios and get a nice clean URL.

**Comment by yoannchaudet** [comment]
To get the behavior you are after, rename your `mateusabelli` repository to `mateusabelli.github.io`.

This is documented right here: https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites 😀.

**Reply by mateusabelli** [reply]
Hello @yoannchaudet Thank you! now I got the result that I was expecting 😃. I'm sorry that I didn't read the documentation before starting this discussion.
